phoenix/app/apiProducts/lib/apiProducts/index_product.ex

```diff
@@ -14,13 +14,25 @@ defmodule ApiProducts.IndexProduct do
         price: product.price
         date: DateTime.to_iso8601(DateTime.utc_now())
       }
-  end
 
-  def products(product) do 
     put("/apiProducts/products/#{product_json.id}", product_json)
-  end 
+  end
 
-  def delete_product_by_index(product) do
+  def delete_product(product) do
     delete("/apiProducts/products/#{product.id}")
   end
+
+  def search_product(params) do
+    query = Enum.map_join(params, "*&", fn {k, v} -> "#{k}:#{v}" end)
```

Comment:
Caça com que se tiver dois filtro ("sku=1&name=test"), sejam complementados como um AND